### DISCUSSION TITLE
Openmetric data import

### DISCUSSION THREAD

**Discussion by aamarques** [thanos-io/thanos#5147] [Questions & Answers] [answered]
Hi,
Is it possible to import RRD data (Openmetric formated) into the object store? 

Cheers, 

AM

**Comment by yeya24** [comment, marked as answer]
If it is in openmetric format then maybe you can use promtool to generate new blocks from the openmetrics file and then the sidecar can upload it to the object storage.

**Reply by aamarques** [reply]
Thanks for your time @yeya24  😃 
That's what I'm doing now using a python program (based in this https://gist.github.com/gillg/9687da29d84b8afd1e0e9372344e4c71)  to convert RRD to OpenMetricts and then using the amazing Thanos kit (https://github.com/sepich/thanos-kit).

PS: I took the liberty of leaving the links in my answer as they helped a lot nad should help someone else 🙏
